packages/integration-tests/src/tests/api/logto-config.test.ts

```diff
@@ -98,5 +96,15 @@ describe('admin console sign-in experience', () => {
       // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
       { id: expect.any(String), signingKeyAlgorithm: 'RSA', createdAt: expect.any(Number) },
     ]);
+
+    const privateKeys2 = await rotateOidcKeys('private-keys', SupportedSigningKeyAlgorithm.RSA);
+
+    expect(privateKeys2).toHaveLength(2);
+    expect(privateKeys2).toMatchObject([
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      { id: expect.any(String), signingKeyAlgorithm: 'RSA', createdAt: expect.any(Number) },
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      { id: expect.any(String), signingKeyAlgorithm: 'EC', createdAt: expect.any(Number) },
```

Comment:
can compare the value of `privateKeys2[1]` and `privateKeys[0]`?

Reply:
We don't return private key values in the API response. But I think we can compare the ID

Reply:
Updated